src/features/nft/portfolio/description/Description.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { TNFT } from 'features/nft/Nft.types'
+import { useToggle } from 'react-use'
+import cn from 'classnames'
+import style from 'features/nft/portfolio/description/Description.module.css'
```

Comment:
@CrazyOptimist we can use './Description.module.css' instead of 'feature....'

Reply:
Yes, it's better :)

Reply:
fixed